frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1626,6 +1651,38 @@ class Worksheet extends React.Component {
         this.setState({ uploadAnchor: e.currentTarget });
     };
 
+    showNewImage = (e) => {
```

Comment:
Can we rename this? It seems like this maybe should be called `uploadNewImage` instead.

Reply:
solved